.circleci/config.yml

```diff
@@ -63,9 +63,66 @@ jobs:
                 path: ~/project/dapp/junit
             - store_artifacts:
                 path: ~/project/dapp/junit
+    
+    getline_ts_build:
+        docker:
+            - image: getline/ci-frontend:latest
+        working_directory: ~/project/getline.ts
+        steps:
+            - checkout:
+                path: ~/project
+            - restore_cache:
+                keys:
+                    - getlinets2-modules-{{ checksum "yarn.lock" }}
+                    - getlinets2-modules
+            - run: yarn
+            - save_cache:
+                key: getlinets2-modules-{{ checksum "yarn.lock" }}
+                paths:
+                    - node_modules/
+            - run:
+                command: yarn generate
+            - run:
+                command: yarn tsc
+            - run:
+                command: mkdir -p /tmp/workspace/getline.ts && cp -rv dist node_modules /tmp/workspace/getline.ts/
```

Comment:
`npm link` might be useful. That way we'll be able to see if getline.ts works as a publishable package and front-end works separately, just installing getline.ts as a dependency.

https://docs.npmjs.com/cli/link

Reply:
I'll keep this in mind for when we're ready to publish it on NPM. It doesn't seem to simplify the mess that is present here with workspace sharing..?